readme.md

```diff
@@ -12,7 +12,7 @@
 - Compare code examples in Vanilla JS and jQuery.
 - Define `$(document).ready()` and some jQuery methods.
```

Comment:
Suggest changing to:

- Use `$(document).ready()` to prevent JS from running before the page has loaded.
- Use jQuery methods to get information from, create, and modify DOM elements